docker-builds/reference-app/Dockerfile

```diff
@@ -9,4 +12,9 @@ ENV ES_URL "https://search-s4s-logs-xsjsafiwd7vkpiucmjqmdjkp7y.us-west-2.es.amaz
 RUN cp development.ini.dist development.ini
 RUN initialize_db development.ini
 
-CMD ["pserve", "development.ini"]
```

Comment:
Can you move the `runserver.sh` contents here -- and better, separate out `pip install` so it runs at build time, and the remaining steps (run `cron`; run `pserve`) into a Docker-run startup script checked in next to this dockerfile?